scheduler-server/sql/scheduler/tables/scheduling_request.sql

```diff
@@ -7,6 +7,7 @@ create table scheduling_request (
   status status_t not null default 'pending',
   reason jsonb null,
   canceled boolean not null default false,
+  dataset_id integer,
```

Comment:
Your migrations checks are failing because you need to either specify `default null` here or remove `default null` from the migration.